source/imaer-gml/src/main/java/nl/overheid/aerius/gml/v5_1/togml/GMLVersionWriterV51.java

```diff
@@ -122,7 +123,7 @@ private SituationMetadata getSituation(final MetaDataInput input) throws AeriusE
       situation = new SituationMetadata();
       situation.setName(input.getName());
       situation.setReference(input.getReference());
-      situation.setSituationType(input.getSituationType());
+      situation.setSituationType(input.getSituationType() != null ? LegacySituationType.safeValueOf(input.getSituationType().name()) : null);
```

Comment:
When writing 5.1, I don't think we should use `LegacySituationType` at all.

Reply:
agreed